dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift

```diff
@@ -78,7 +78,45 @@ class SingaporeUserHappyFlowWithScanICStageDeciderTests: XCTestCase {
         
         XCTAssertEqual(decider.compute(context: context, localContext: localContext), .address)
     }
-    
+
+    func testUserHasCompletedJumioFailedCallback() {
+        let decider = StageDecider()
+        let localContext = LocalContext(selectedRegion: Region(id: "sg", name: "SG"), hasSeenNotificationPermissions: true, regionVerified: true, hasSeenVerifyIdentifyOnboarding: true, selectedVerificationOption: IdentityVerificationOption.scanIC, hasCompletedJumio: true, hasSeenLocationPermissions: true)
+
+        let context = Context(
+            customer: CustomerModel(id: "xxx", name: "xxx", email: "xxxx@gmail.com", analyticsId: "xxxx", referralId: "xxxx"),
+            regions: [
+                RegionResponse(
+                    region: Region(id: "sg", name: "Singapore"),
+                    status: .PENDING,
+                    simProfiles: nil,
+                    kycStatusMap: KYCStatusMap(jumio: .REJECTED, myInfo: .PENDING, nricFin: .APPROVED, addressPhone: .APPROVED)
+                )
+            ]
+        )
+
+        XCTAssertEqual(decider.compute(context: context, localContext: localContext), .ohNo(.ekycRejected))
+    }
+
+    func testUserHasCompletedJumioFailedAndClearedLocally() {
```

Comment:
This name is unclear. What scenario are you testing?

Reply:
Forget this, I am removing this PR